packages/loader/container-loader/src/loader.ts

```diff
@@ -336,6 +336,21 @@ export class Loader implements IHostLoader {
 			clientDetailsOverride?: IClientDetails;
 		},
 	): Promise<IContainer> {
+		const container = this.createDetachedContainerUninitialized(
+			codeDetails,
+			createDetachedProps,
+		);
+		//*
+		return container.initialize().then(() => container);
+	}
+
+	public createDetachedContainerUninitialized(
```

Comment:
we should be moving away from the loader, so i wouldn't expect any changes in the loader, the functions should do all the work